system/jlib/jflz.cpp

```diff
@@ -52,6 +52,8 @@
 
 #include "jcrc.hpp"
 
+#include "jlog.hpp"
+
```

Comment:
I added it for some DBGLOG statements that I've since removed.  I will remove this include.